code/modules/1713/language_book.dm

```diff
@@ -56,7 +56,7 @@
 		langcopy.Remove(language1)
 		var/datum/language/language2 = WWinput(user, "What will your 2nd language to translate be?", "Writing Translations", "text", langcopy)
 		user.visible_message("<span class='notice'>[user] begins to write in the [src].</span>", "<span class='notice'>You begin writing translations in the [src]. This will take around 2 minutes, and you need to stand or sit still.</span>")
-		if(do_after(user, 1200, src))
+		if(do_after(user, 1800, src))
```

Comment:
You made it longer to write?

Reply:
i somehow failed to remove that 1 before it apparently lmao shit

Reply:
somehow messed up while reverting it to original time to write